src/Microsoft.Bcl.AsyncInterfaces/pkg/Microsoft.Bcl.AsyncInterfaces.pkgproj

```diff
@@ -6,5 +6,10 @@
     </ProjectReference>
     <ProjectReference Include="..\src\Microsoft.Bcl.AsyncInterfaces.csproj" />
   </ItemGroup>
+  <PropertyGroup>
+    <!-- Excluding reference assets so as to not expose reference assemblies for Desktop projects that use the
+         workflow compiler. See https://github.com/dotnet/runtime/issues/37496 for more details  -->
+    <ExcludeReferenceAssets>true</ExcludeReferenceAssets>
```

Comment:
Can we just omit the ProjectReference instead?

Reply:
Sure.